tests/rptest/tests/raft_availability_test.py

```diff
@@ -222,6 +222,22 @@ def test_one_node_down(self):
         the cluster remains available afterwards, and that the expected
         peer takes over as the new leader.
         """
+        # Retry a couple of times to establish new leader in a single election.
+        # This is racy if we just run it once. Running it multiple times
+        # gives us more confidence to actually detect a regression.
+        # Ref: https://github.com/vectorizedio/redpanda/issues/2444
+        retries = 4
+        ex: Exception = None
+        for i in range(1,retries+1):
+            try:
+                self._one_node_down()
+                return
+            except MetricCheckFailed as ex:
+                self.logger.info("Failed to elect new leader in single election, retrying "
+                    + "attempt ({i} of {retries}")
+        raise ex
+
+    def _one_node_down(self):
         # Find which node is the leader
         initial_leader_id, replicas = self._wait_for_leader()
         assert initial_leader_id == replicas[0]
```

Comment:
This assertion fails on a retry, because the system is in whatever state it was at the end of the previous iteration, rather than in the pristine first-start state.  Maybe need a boolean argument to this function for whether it's the first run or not -- I'd rather not lose this assertion.